django_react_proj/users-fe/src/components/NewUserForm/NewUserForm.js

```diff
@@ -11,29 +12,32 @@ const NewUserForm = ({
 }) => {
   const [name,setName]=useState(user?.name || "");
   const [password,setPassword]=useState(user?.password || "");
+  const [deadline,setDeadline]=useState(user?.deadline || "");
 
   const onNameChange = event => {
     setName(event.target.value)
   };
   const onPasswordChange = event => {
     setPassword(event.target.value)
   };
-
+  const onDeadlineChange = event => {
+    setDeadline(event.target.value)
+  };
   const defaultIfEmpty = value => {
     return value === "" ? "" : value;
```

Comment:
This kinda makes no sense. If you want the empty string if the value is an empty string or value otherwise you can just use `return value`.

Reply:
Now when Arczi4 upload new commit with variable "deadline" in db i think it makes sense.